[2023-10-07 08:24:49,157] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sales_mart.update_d_customer manual__2023-10-07T08:23:34.489254+00:00 [queued]>
[2023-10-07 08:24:49,163] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sales_mart.update_d_customer manual__2023-10-07T08:23:34.489254+00:00 [queued]>
[2023-10-07 08:24:49,164] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-07 08:24:49,165] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-07 08:24:49,166] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-07 08:24:49,174] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): update_d_customer> on 2023-10-07 08:23:34.489254+00:00
[2023-10-07 08:24:49,177] {standard_task_runner.py:52} INFO - Started process 2138 to run task
[2023-10-07 08:24:49,178] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'sales_mart', 'update_d_customer', 'manual__2023-10-07T08:23:34.489254+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/sprint3.py', '--cfg-path', '/tmp/tmpa2ywd59p', '--error-file', '/tmp/tmptrk85muj']
[2023-10-07 08:24:49,179] {standard_task_runner.py:80} INFO - Job 57: Subtask update_d_customer
[2023-10-07 08:24:49,217] {task_command.py:369} INFO - Running <TaskInstance: sales_mart.update_d_customer manual__2023-10-07T08:23:34.489254+00:00 [running]> on host 36d9c57a9c84
[2023-10-07 08:24:49,257] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=student@example.com
AIRFLOW_CTX_DAG_OWNER=student
AIRFLOW_CTX_DAG_ID=sales_mart
AIRFLOW_CTX_TASK_ID=update_d_customer
AIRFLOW_CTX_EXECUTION_DATE=2023-10-07T08:23:34.489254+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-10-07T08:23:34.489254+00:00
[2023-10-07 08:24:49,264] {base.py:68} INFO - Using connection ID 'postgresql_de' for task execution.
[2023-10-07 08:24:49,268] {sql.py:315} INFO - Running statement: insert into mart.d_customer (customer_id, first_name, last_name, city_id)
select customer_id, first_name, last_name, max(city_id) from staging.user_order_log
where customer_id not in (select customer_id from mart.d_customer)
group by customer_id, first_name, last_name, parameters: None
[2023-10-07 08:24:49,270] {sql.py:324} INFO - Rows affected: 0
[2023-10-07 08:24:49,279] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=sales_mart, task_id=update_d_customer, execution_date=20231007T082334, start_date=20231007T082449, end_date=20231007T082449
[2023-10-07 08:24:49,312] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-07 08:24:49,337] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
